demisto_sdk/commands/common/constants.py

```diff
@@ -858,6 +858,19 @@ class FileType(Enum):
     'doc_imgs',
 ]
 
+VALIDATION_USING_GIT_IGNORABLE_DATA = (
+    'Pipfile',
+    'Pipfile.lock',
+    'command_examples',
```

Comment:
Im pretty worried about  this one. we do not enforce the name of the example file and we have various names on the repo.
see the following for examples:
Packs/SentinelOne/Integrations/SentinelOne-V2/examples.txt
Packs/BitSight/Integrations/BitSightForSecurityPerformanceManagement/command-examples.txt